relnotes/taskpool.feature.md

```diff
@@ -0,0 +1,4 @@
+* `ocean.task.TaskPool`
+
+  A new function `awaitAllTasks()` has been added. This function suspends the
+  current task until all tasks in the pool have finished executing.
```

Comment:
How about: "This function suspends the current task until all tasks in the pool have finished executing. It is assumed that the current task (the one being suspended) is _not_ a task in the pool." ?

Maybe add that to the function doc, as well. (I also wonder about asserting that this is the case, but I'm not sure if there's a way to do that without iterating over the whole pool, which seems a bit excessive.)